website/pages/docs/plugins/sources/gcp/configuration.md

```diff
@@ -80,7 +80,7 @@ This is the (nested) spec used by GCP Source Plugin
 
 - `scheduler` (string) (default: `dfs`):
 
-  The scheduler to use when determining the priority of resources to sync. Currently, the only supported values are `dfs` (depth-first search) and `round-robin`. This is an experimental feature, and may be removed in the future. For more information about this, see [performance tuning](/docs/advanced-topics/performance-tuning).
+  The scheduler to use when determining the priority of resources to sync. Currently, the only supported values are `dfs` (depth-first search), `round-robin` and `shuffle`. For more information about this, see [performance tuning](/docs/advanced-topics/performance-tuning).
```

Comment:
Add the info to corresponding `_configuration.mdx` file as well?

Reply:
But it currently doesn't contain anything regarding these advanced options...

```yaml copy
kind: source
spec:
  # Source spec section
  name: "gcp"
  path: "cloudquery/gcp"
  version: "VERSION_SOURCE_GCP"
  tables: ["gcp_storage_buckets"]
  destinations: ["DESTINATION_NAME"]
  spec:
    # GCP Spec section described below
    project_ids: ["my-project"]
```

Reply:
Yeah I think bringing all the parameters into `_configuration.mdx` shouldn't fall under the scope of this PR